build.gradle

```diff
@@ -193,6 +193,7 @@ configure(javaProjects) {
 
         // Testing utilities
         testCompile project(':testing')
```

Comment:
We don't need this line anymore because `testing-internal` depends on it, right?